src/chart-review/appointments/appointments.component.tsx

```diff
@@ -0,0 +1,23 @@
+import { useRouteMatch } from "react-router-dom";
+import ChartWidget from "../../ui-components/chart-widget/chart-widget.component";
+import React from "react";
+import AppointmentsOverview from "../../widgets/appointments/appointments-overview.component";
+
+export default function Appointment(props: any) {
+  const match = useRouteMatch();
+
+  const widgetConfig = {
```

Comment:
we can wait for #129 to be merged so that you can utilize the config module

Reply:
Does this mean all the other widgets will have to be refactored to use the config module? If so why don't we merge this then we can have a single ticket to gather for the refactor of all widgets?

Reply:
We're still figuring out this. In general these widgets need to be coded such that they have no dependencies outside of there repo with the one exception being 'react-router-dom'. That will be a webpack external and thus you can continue to use it's api.